doc/dev/developer_guide/essentials.rst

```diff
@@ -7,9 +7,9 @@ to know.
 Leads
 -----
 
-The Ceph project is led by Sage Weil. In addition, each major project
-component has its own lead. The following table shows all the leads and
-their nicks on `GitHub`_:
+The Ceph project was created by Sage Weil and is led by the Ceph Leadership
+Team (CLT). In addition, each major project component has its own lead. The
+following table shows all the leads and their nicks on `GitHub`_:
```

Comment:
Venky Shankar is now the lead of CephFS.

Reply:
I will merge this tomorrow, when I have opened a PR that updates the CephFS lead from Patrick Donnelly to Venky Shankar.